.gitmodules

```diff
@@ -3,7 +3,8 @@
 	url = https://github.com/NCAR/TEMPO.git
 [submodule "src/core_atmosphere/physics/UGWP"]
 	path = src/core_atmosphere/physics/UGWP
-	url = https://github.com/mdtoyNOAA/UGWP.git
+	url = https://github.com/NOAA-GSL/UGWP.git
+        branch = main_knob_ugwp_tauamp_namelist
```

Comment:
I thought we were only supposed to point to a "stable" branch. Is this just temporary?

Reply:
Can you remind me how we define a "stable" branch?  I assume it means one that will never change, like the "main" branch does.  I don't plan on changing this branch.  Should I instead address it by the hash and not the name?

Reply:
Yes, something like a "main" branch that is considered to be the authoritative branch that collects commits from all other developmental branched. If that it your "main", then I suppose it's OK. I just wondered. I wonder if we should all specify the name of our authoritative branch(?).